src/Microsoft.Health.Fhir.Core/Features/Search/Registry/SearchParameterStatusManager.cs

```diff
@@ -58,6 +58,8 @@ internal async Task EnsureInitializedAsync(CancellationToken cancellationToken)
             var parameters = searchParamResourceStatus.ToDictionary(x => x.Uri);
             _latestSearchParams = parameters.Values.Select(p => p.LastUpdated).Max();
 
+            _logger.LogInformation("There are '{SearchParameterCount}' SearchParameters", searchParamResourceStatus.Count.ToString());
```

Comment:
Unless you are tracking something other than the count of custom search parameters, such as whether the count in this manager matches the definition manager, you don't need this one.